src/vs/platform/native/electron-main/nativeHostMainService.ts

```diff
@@ -687,6 +687,17 @@ export class NativeHostMainService extends Disposable implements INativeHostMain
 	//#endregion
 
 
+	//#region Screenshots
+
+	async getScreenshot(windowId: number | undefined): Promise<ArrayBufferLike | undefined> {
+		const window = this.windowById(undefined, windowId);
```

Comment:
Do we need to account for auxiliary windows? In the end, chat might be in an aux window and then using the primary window `webContents` could be wrong?

Reply:
I'll check how it's handled there, I guess almost always at least in this initial use case we want the screenshot of the primary window.

Reply:
Handling it the same as most users of `windowById`: https://github.com/microsoft/vscode/pull/231410